website/settings/defaults.py

```diff
@@ -588,6 +595,10 @@ class CeleryConfig:
                 'task': 'scripts.generate_sitemap',
                 'schedule': crontab(minute=0, hour=5),  # Daily 12:00 a.m.
             },
+            'check_crossref_doi': {
+                'task': 'management.commands.check_crossref_dois',
```

Comment:
Forgot one thing -- this needs to be added to `CeleryConfig.imports`

Reply:
👍